doc/Type/Metamodel/CurriedRoleHOW.pod6

```diff
@@ -61,6 +61,10 @@ a single parameter an instantiated role will also be of the same type:
     role Zape[::T] {};
     say Zape[Int].HOW; #: «Perl6::Metamodel::CurriedRoleHOW.new␤»
 
+Curried roles L<pretend to be|/type/Metamodel::TypePretense> of types
+C<Mu>, C<Any>, and C<Cool>, and
+L<delegate|/type/Metamodel::MethodDelegation> methods to C<Any>.
+
 I<Note>: As most of the C<Metamodel> classes, this class is here mainly for
 illustration purposes and it's not intended for the final user to instantiate.
 
```

Comment:
Know it's not part of the PR, but I don't see a point in this sentence any more. Metamodels are being used extensively in the wild now, from RED to other modules...

Reply:
I agree. `Metamodel::ClassHOW` in particular is something that's useful to subclass and is something I subclass in `Data::Record`. One of the module's kinds, `MetamodelX::RecordTemplateHOW`, behaves similarly to role groups, but I don't subclass `Metamodel::ParametricRoleGroupHOW` for those since they produce `MetamodelX::RecordHOW`s, not roles. I can see it being useful to subclass the role HOWs in certain cases, though.